src/Views/Components/ListTable/ListTable/index.tsx

```diff
@@ -184,6 +185,7 @@ export const ListTable = ({
                             </tr>
                         </thead>
                         <tbody className={styles.tableContent}>
+                            {window.GiveDonations && <ProductRecommendation columns={visibleColumns.length + 1} />}
```

Comment:
I'm a bit confused by this. Is `window.GiveDonations` being used to render it only for the donations table? I'd like to go about this different, if so, as to not have this so explicitly coupled to that window property. Ideally, the ListTable would receive the `ProductRecommendation` as an optional prop and be used here if it is passed.

Reply:
I refactored this with some suggestions from Jon to make the component and the request handling more flexible. Resolved:https://github.com/impress-org/givewp/pull/6797/commits/0ef9b08e281a85c221bcc1dbae1857e4e327929e +  https://github.com/impress-org/givewp/pull/6797/commits/01085c10090172489c9593631fe6995d277c9d96